### ISSUE TITLE
utils.importMap command fails to load

### ISSUE THREAD

**Issue by tiennou** [ScreepsMods/screepsmod-admin-utils#36]
I found this error in `engine_main.log` after wondering why'd I get the default map instead of a random 2x2:

```
Error loading "/screeps/mods/node_modules/screepsmod-admin-utils/index.js": Error: Cannot find module 'fs/promises'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/screeps/mods/node_modules/screepsmod-admin-utils/lib/services/importMap/utils.js:3:19)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
```

*EDIT*: This is while using Jomik's Docker server. Starting it is enough to get the error to pop up

**Comment by AlinaNova21** [comment]
Node version is likely too old, expects at least node 12 iirc

**Comment by tiennou** [comment]
That has been fixed by #37
